stacks/serverless/dovetail-counts-lambda.yml

```diff
@@ -125,3 +128,115 @@ Resources:
       EventSourceArn: !Ref KinesisIn
       FunctionName: !Ref DovetailCountsLambdaFunction
       StartingPosition: "LATEST"
+  # log metrics
+  DovetailCountsLambdaWarnsMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '{ $._logLevel = "warn" }'
+      LogGroupName: !Sub "/aws/lambda/${DovetailCountsLambdaFunction}"
```

Comment:
Is this going to work?

I also thought about specifically defining the `AWS::Logs::LogGroup` for the lambda in here.  (So I could also add an expiration).  But if I recall, you can't add existing log groups to CFN without some monkeying around.  Pausing the lambda, deleting the log group, etc.

Reply:
Should work since the log group already exists. I'm not sure what would happen if this were a virgin stack and you tried to make a metric filter for a log group that didn't exist

Reply:
It should be fine since the log group already exists. I'm not sure what happens if you try to make a metric filter for a nonexistent log group.